src/masternodes/govvariables/attributes.h

```diff
@@ -93,7 +98,7 @@ enum DFIPKeys : uint8_t {
     MintTokens           = 't',
     EVMEnabled           = 'u',
     ICXEnabled           = 'v',
-    AllowDUSDLoops       = 'w',
+    DUSDVault            = 'w',
```

Comment:
Should probably have its own entry under VaultKeys. DFIPKeys are used for AttributeTypes::Param. That's how we would normally add a new one to scale for additional uses later.

Reply:
Sure will make the changes, i thought of leaving it here since the implementation is a special DFIP.

Reply:
Resolved in commit [7b343b7](https://github.com/DeFiCh/ain/pull/2187/commits/7b343b7661a5e7faa810002a90fc3d91806d991f)